content/en/docs/about/_index.md

```diff
@@ -3,3 +3,9 @@ title = "About"
 description = "Information about Kubeflow software, community, docs, and events."
 weight = 10
 +++
+
+{{% alert title="Out of date" color="warning" %}}
```

Comment:
The About Kubeflow section doesn't appear to have version specific content. Would it be possible to exclude this section?